Config.uk

```diff
@@ -180,6 +180,13 @@ if RECORD_BUILDTIME
 	endchoice
 endif
 
+config BINARY_COMPAT
```

Comment:
Can't we provide an option with `vfscore` instead? This has to do with largefile support, so I'd say to add an option there instead that enables the `64` bits or `32` bits version of the system calls (and also registering those variants with `UK_LLSYSCALL_R_DEFINE()`). The key would be to keep the behavior as is when we natively compile (e.g., via preprocessor renaming tricks as of today) and only get either the old style or new style system call handlers registered when the option is off. With `elfloader` we would enable this option in vfscore for binary compatibility. Alternatively, the default of this option in vfscore could be `y` if binary system calls are enabled in syscall_shim:
```KConfig
default n
default y if `LIBSYSCALL_SHIM_HANDLER`
```